desktop/core/src/desktop/auth/views.py

```diff
@@ -100,7 +100,7 @@ def dt_login_old(request, from_modal=False):
 
 
 @login_notrequired
-@watch_login
+@axes_dispatch
```

Comment:
because of this error `ImportError: cannot import name 'watch_login' from 'axes.decorators'`

Reply:
found this relevant to this problem https://github.com/jazzband/django-axes/issues/346

Reply:
Interesting and indeed we need to find the equivalent for tracking the login failures now!

Reply:
And then this should be the last changes to do